src/app/locale/translations/en.json

```diff
@@ -274,8 +274,8 @@
             "account_instrument_new_shipping_address": "<strong>We noticed this is a new shipping address.</strong><p>For security reasons, you will need to re-link your PayPal account when shipping to an address for the first time or if the shipping address was edited recently.</p>",
             "instrument_trusted_shipping_address_text": "This additional security step is applied to your card when shipping to an address for the first time or if the shipping address was edited recently.",
             "instrument_trusted_shipping_address_title_text": "Please re-enter your card number to authorize this transaction.",
-            "quadpay_continue_action": "Continue with Quadpay",
-            "quadpay_display_name_text": "Pay in 4 installments",
+            "quadpay_continue_action": "Continue with Zip",
+            "quadpay_display_name_text": "Pay in 4 installments (Quadpay)",
```

Comment:
If they're rebranding to Zip, why does this say Quadpay?

Reply:
Product's requirement:

![Screen Shot 2021-08-24 at 14 13 53](https://user-images.githubusercontent.com/4503787/130676162-b3422864-ab71-40ac-8538-a5f67cf4776c.png)